### ISSUE TITLE
Django Admin TabularInline "Add another"-row not select2

### ISSUE THREAD

**Issue by nubtwo** [asyncee/django-easy-select2#52]
Django version 1.11.13
django-easy-select2 1.5.3

The fields of an existing row of a TabularInline work with select2, 
if I click "Add another",  the fields of the new row are not select2:

![grafik](https://user-images.githubusercontent.com/16845311/43959762-32d29114-9cb0-11e8-8975-6d8550c993f6.png)

Ideas?

**Comment by gauravv7** [comment]
Hi @nubtwo , apologies for replying soo late.

I checked this issue and confirmed this. Long awaited deadline has now come, "DomNodeInserted" event is not supported by chrome now I guess.
I tried running a similar config project for any bugs in current code so the same scenario described as above works in mozilla(all browsers majorly) except my chrome (would have blamed browsers in case of other browsers but chrome makes me feel now this is my fault haha,.. ).

so I'll try to wind up releasing a new version in a week or 2. need some performance and testing to be done for the changes but the basic idea is to use Mutation Observer.
here is the code for DomNodeInserted replacement, 

just copy paste it(currently tested for above scenario only) but should work for all cases.

https://gist.github.com/gauravv7/a994fb05dbee9ccc828423f605654327.js

Will update surely for a proper release with easy-select2.

Let me know if I can help more,
Thanks

**Comment by asyncee** [comment]
Closed via #54

**Comment by nubtwo** [comment]
I am sorry, but my problem in my first post remains. 
The $(document).on('formset:added', function (event, $row, formsetName) never gets hit.
Any ideas? How can I trace the origin of the error?

**Comment by gauravv7** [comment]
yes, confirming this issue now, actually formset:added is related to django.JQuery which is included in our plugin as a JQuery port, but since easy-select2 was initially meant to be a simple wrapper around select2.js , it uses jquery and not Django.JQuery (as simply relating to docs of select2), and therefore not getting the event launch from Django.JQuery.

@nubtwo and @asyncee  the fix wil be provided today itself.

**Comment by asyncee** [comment]
Thanks to @gauravv7 we have new release 1.5.5 on pypi.

@nubtwo would be you so kind to test and confirm if problem is solved for you?

**Comment by nubtwo** [comment]
@gauravv7 @asyncee I can confirm it works now! Django version 1.11.13.

Thank you very much. You guys are so fast. Where is the donation button?
Greetings from Germany.

**Comment by asyncee** [comment]
We do not have one :)

**Comment by gauravv7** [comment]
should I close this thread? because I don't want to :D
